### PR TITLE
Hello-World

### PR DESCRIPTION
**Pull request by shurichmond**
### A Simple HTML and CSS project. 

- [x] Added an H1 tag for the heading with a class
- [x] Added a CSS style of color to it

### CODE REVIEW

**Review by skyv26 — CHANGES_REQUESTED** [review]
## Hi @shurichmond , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct `GitHub` flow ✅ 
- Well-implemented requirements ✅ 

## Required Changes ♻️

Congratulations 🎉  on the impressive work you've done so far. I've made some suggestions below to help you improve your project so kindly take a look at it and if you don't understand anything please don't hesitate to ask. Best of luck! 🤞🏻. Using inline comments would have been the best but since I don't have access to your `README.md` and `.gitignore` in this pull request. I will do my best to explain the changes here. Let's get started.

- [ ] Please always remove the placeholders from your `README.md` because placeholders are for guidance purposes for developers while creating the `documentation` for the project to showcase your project, the project's features and many other things. Having these kinds of placeholders makes your `README.md` file looks unprofessional. Kindly have a look at the below-attached image.

Screenshot from 2023-04-28 11-55-19

- [ ] Please adhere to the project requirements according to which your `test.md` file should be ignored by the `.gitignore` file. No doubt you did add the `test.md` file but you added it after committing your `test.md` file. I would suggest you 3 steps to solve the issue.
  - Please delete the `test.md` file and make a commit.
  - Please add `test.md` file back again with the required content.
  - when you try to finally push your code you will see it will be ignored by `.gitignore`.

Screenshot from 2023-04-28 11-56-34


**Caution:** *Please don't touch .gitignore file*


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @skyv26 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ If you like this project, You can support it by giving it a star ⭐
 
 > Give credit to everyone who inspired your codebase.
 
-I would like to thank...
+I would like to thank My friends for the support and inspiration.
```
Comment (ABDULALI3468) [inline README.md:130]: As requested by the previous code reviewer please consider removing the commented line as it has nothing to do with the user who is reading your README else it was there just to guide you when writing a professional README.

image


I am talking about the following text:-

`Give credit to everyone who inspired your codebase.`

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @shurichmond , 



Great work on making the changes requested by a previous reviewer 👏 
You've done really well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — COMMENTED** [review]
Hi @shurichmond, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Alt Text

 ## Highlights


 ## Required Changes ♻️

- [ ] Please note that in order for your project to be `reviewed` properly, it's important that you do not `merge` the pull request until it has been `approved`. Pull requests are opened specifically for the purpose of `review` and should not be `merged` until all feedback has been addressed and the changes have been `approved`. This will help ensure that the final product is of high quality and meets all requirements.

- [ ] As requested by the previous reviewers kindly implement the changes suggested [here](https://github.com/shurichmond/Hello-World/pull/3/files#r1180228548) before resubmitting your project for review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**